src/pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,122 @@
+import { useEffect, useMemo, useState } from 'react';
+import { Loader } from '../../components/Loader';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { ErrorMessage } from '../../components/ErrorMessage/ErrorMessage';
+import { PeopleFilters } from '../../components/PeopleFilters/PeopleFilters';
+import { useSearchParams } from 'react-router-dom';
+import { getSearchWith } from '../../utils/searchHelper';
+import { getFilteredPeople } from '../../utils/getFilteredPeople';
+import { PeopleTable } from '../../components/PeopleTable/PeopleTable';
+import { SearchParams } from '../../types/SearchParams';
+import { SortField, SortOrder } from '../../types/SortTypes';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [errorMessage, setErrorMessage] = useState(false);
+
+  const fetchPeople = async () => {
+    setIsLoading(true);
+    try {
+      const result = await getPeople();
+
+      setPeople(result);
+    } catch (error) {
+      setErrorMessage(true);
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  useEffect(() => {
+    fetchPeople();
+  }, []);
+
+  // SEARCH PARAMS
+
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const query = searchParams.get('query') || '';
+  const centuries = searchParams.getAll('centuries') || null;
+  const sex = searchParams.get('sex') || '';
+  const sort = (searchParams.get(SearchParams.sort) as SortField) || null;
+  const order =
+    (searchParams.get(SearchParams.order) as SortOrder) || SortOrder.Asc;
+
+  const handleQueryChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const appliedQuery = e.target.value.trim() ? e.target.value : null;
```

Comment:
```suggestion
    const appliedQuery = event.target.value.trim() || null;
```
Please avoid one letter naming